gen/definitions/virtual_network_to_fabric_site.yaml

```
@@ -0,0 +1,41 @@
+---
+name: Virtual Network To Fabric Site
+rest_endpoint: /dna/intent/api/v1/business/sda/virtual-network
+no_update: true
+no_import: true
+no_data_source: true
+skip_minimum_test: true
+res_description: Add virtual network (VN) to SDA Fabric Site
```

It seems to be able to delete as well as add so maybe something like:

```suggestion
res_description: Manage assignment of a virtual network (VN) to an SD-Access Fabric Site.
```

---

sounds good, changed that